docs/UserGuide.md

```diff
@@ -167,8 +168,8 @@ A command line has 4 components, **command word**, **index**, **prefix** and **p
 | Parameter    | `John Doe` | Represents the updated value or content. <br/> These values, which are provided by the user, replaces the `UPPER_CASE` placeholders in the command format |
```

Comment:
I thought u wanted to edit the description?

Reply:
![image](https://github.com/AY2324S1-CS2103T-T17-2/tp/assets/108869153/19c5a300-bae1-4ca6-ac4d-2e960b4f81bf)

i was actually referring to the `John Doe` part. shorten to John?